content/applications/sales/sales/invoicing/time_materials.rst

```diff
@@ -2,135 +2,352 @@
 
-It can then be submitted to the manager, approved and finally posted.
+First, add a :guilabel:`Vendor` to the purchase order. Then, under the :guilabel:`Products` tab,
+click the :guilabel:`extra column options` drop-down menu, represented by two horizontal lines with
+dots on them, located to the far-right of the column headers. From that drop-down menu, select
+:guilabel:`Analytic Distribution`.
 
-.. image:: time_materials/time_materials10.png
+.. image:: time_materials/extra-column-analytic-distribution-option.png
    :align: center
+   :alt: How to add analytic distribution column on purchase order form in Odoo Purchase.
 
-.. image:: time_materials/time_materials11.png
+After adding the :guilabel:`Analytic Distribution` column to the headers on the :guilabel:`Products`
+tab of the purchase order form, proceed to add the product(s) to the purchase order. To do that,
+click :guilabel:`Add a product`, and select the desired product from the drop-down menu. Repeat for
+all the products to add.
+
+.. important::
+   In order for a purchase to be properly invoiced on a sales order, the product on the purchase
+   order **must** be marked as :guilabel:`Can be Expensed`, have an :guilabel:`Invoicing Policy` set
+   to :guilabel:`Delivered quantities` and the :guilabel:`At cost` option selected in the
+   :guilabel:`Re-Invoice Expenses` field on its product form.
+
+   .. image:: time_materials/product-form-settings-invoice-purchase.png
+      :align: center
+      :alt: Product settings for a purchase order to be invoiced on a sales order in Odoo.
+
+Then, select the appropriate :guilabel:`Analytic Distribution` associated with the sales order to
+which this purchase order is related. To do that, click the empty :guilabel:`Analytic Distribution`
+field to reveal an :guilabel:`Analytic` pop-up window.
+
+Then, from the :guilabel:`Departments` drop-down menu select the analytic distribution associated
+with the desired sales order to be invoiced for the purchase.
+
+.. image:: time_materials/analytic-drop-down-distribution.png
    :align: center
+   :alt: How to select the Analytic Distribution department from a purchase order in Odoo.
+
+Once all the information is entered in the :guilabel:`Products` tab of the purchase order, proceed
+to confirm the order by clicking :guilabel:`Confirm Order`. Then, click :guilabel:`Receive
+Products`.
+
+.. note::
+   If any serial/lot numbers must be entered before validation, click the :guilabel:`details` icon,
+   represented by four horizontal lines, located to the far-right of the product line to reveal a
+   :guilabel:`Detailed Operations` tab, in which the necessary :guilabel:`Lot/Serial Number(s)` and
+   :guilabel:`Done` quantity can be added.
 
-.. image:: time_materials/time_materials12.png
+When ready, click :guilabel:`Confirm` to confirm the data and return to the purchase order. Then,
+click :guilabel:`Validate` to validate the purchase order.
```

Comment:
Moved the sentence about confirming up into the Note, since it's a continuation of that flow. Also, we're validating the receipt form, not the purchase order (we already confirmed the purchase order).
```suggestion
Then,
click :guilabel:`Validate` to validate the receipt.
```